src/ex.rs

```diff
@@ -75,36 +76,47 @@ pub struct ExOpts {
     pub crates: ExCrateSelect,
 }
 
-pub fn define(opts: ExOpts) -> Result<()> {
+pub fn define(opts: ExOpts, config: &Config) -> Result<()> {
     delete(&opts.name)?;
     let crates = match opts.crates {
         ExCrateSelect::Full => lists::read_all_lists()?,
-        ExCrateSelect::Demo => demo_list()?,
+        ExCrateSelect::Demo => demo_list(config)?,
         ExCrateSelect::SmallRandom => small_random()?,
         ExCrateSelect::Top100 => top_100()?,
     };
     define_(&opts.name, opts.toolchains, crates, opts.mode)
 }
 
-fn demo_list() -> Result<Vec<Crate>> {
-    let demo_crate = "lazy_static";
-    let demo_gh_app = "brson/hello-rs";
-    let mut found_demo_crate = false;
-    let crates = lists::read_all_lists()?
+fn demo_list(config: &Config) -> Result<Vec<Crate>> {
+    let mut crates = config
+        .demo_crates()
+        .crates
+        .iter()
+        .cloned()
```

Comment:
If we remove this then don't we just collect into a `HashSet<&str>`, which seems fine?